openpype/hosts/maya/plugins/publish/extract_look.py

```diff
@@ -197,10 +204,11 @@ def process(self,
 
         self.log.debug(" ".join(subprocess_args))
         try:
-            run_subprocess(subprocess_args)
+            output = run_subprocess(subprocess_args, logger=self.log)
```

Comment:
I suspect the log of the publisher plug-in might still need to be passed to the `TextureProcessor` [here](https://github.com/ynput/OpenPype/blob/7ed5442f3b4d762b9211664ff7358c9b43635595/openpype/hosts/maya/plugins/publish/extract_look.py#L494) like `Processor(log=self.log)` - or does this already work?

Reply:
`TextureProcessor` is creating its own logger if not passed and that should work too, but passing the one from the plugin is cleaner.